lib/onScreen-rails/version.rb

```diff
@@ -1,5 +1,5 @@
 module OnScreen
   module Rails
-    VERSION = "1.0.0"
+    VERSION = "1.0.1"
```

Comment:
This wouldn't be a patch. Plus, the versioning here should match the library's version: ie `1.2.0`